src/channels/components/ChannelForm/ChannelForm.tsx

```diff
@@ -31,7 +39,10 @@ export interface FormData {
   shippingZonesIdsToRemove: string[];
   warehousesIdsToAdd: string[];
   warehousesIdsToRemove: string[];
+  shippingZonesToDisplay: ChannelShippingZones;
+  warehousesToDisplay: ChannelWarehouses;
   defaultCountry: CountryCode;
+  stockSettings: StockSettingsInput;
```

Comment:
How about `FormData extends StockSettingsInput`?
It'll make form data flat here so we don't have to make weird event spreads ...

Reply:
Great idea, fixed